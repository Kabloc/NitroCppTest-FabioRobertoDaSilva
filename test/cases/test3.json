{
"_comment": "Negative X and Negative Y input",
"rects": [
{"x": 0, "y": 5, "w": 10, "h": 10 },
{"x": -10, "y": 0, "w": 10, "h": 10 },
{"x": -20, "y": -5, "w": 10, "h": 10 },
{"x": -30, "y": 5, "w": 10, "h": 10 },
{"x": -40, "y": 5, "w": 10, "h": 10 },
{"x": 0, "y": -5, "w": 10, "h": 10 },
{"x": -10, "y": -10, "w": 10, "h": 10 },
{"x": -20, "y": -15, "w": 10, "h": 10 },
{"x": -30, "y": -5, "w": 10, "h": 10 },
{"x": -40, "y": -5, "w": 10, "h": 10 }
]
}