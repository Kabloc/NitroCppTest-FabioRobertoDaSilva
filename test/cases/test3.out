Input:
	1: Rectangle at (0, 5), w=10, h=10.
	2: Rectangle at (-10, 0), w=10, h=10.
	3: Rectangle at (-20, -5), w=10, h=10.
	4: Rectangle at (-30, 5), w=10, h=10.
	5: Rectangle at (-40, 5), w=10, h=10.
	6: Rectangle at (0, -5), w=10, h=10.
	7: Rectangle at (-10, -10), w=10, h=10.
	8: Rectangle at (-20, -15), w=10, h=10.
	9: Rectangle at (-30, -5), w=10, h=10.
	10: Rectangle at (-40, -5), w=10, h=10.
